prisma/zod/attendee.ts

```diff
@@ -1,7 +1,6 @@
-import * as z from "zod";
```

Comment:
It seems that the zod schemas got committed without prettier formatting. Try running `yarn generate-schemas` locally to fix this.